src/commands/node/list.ts

```diff
@@ -30,20 +30,24 @@ export default class NodeList extends BlockchainCommand {
       return await nodes.getNodes(operatorId, flags.topology, i, n, { blockTag });
     });
     results = results.filter((v) => {
-      // Filter out disabled nodes if enabledOnly flag is true
-      if (flags.enabled && v.disabled) return false;
+      // Filter out disabled nodes if onlyEnabled flag is true
+      if (flags.onlyEnabled && v.disabled) return false;
+
```

Comment:
this function finally makes sense to me, and it was correct before. just confusing. all i did is change the code to make more sense to me. hopefully it does to you too